AndroidPubSub/README.md

```diff
@@ -149,3 +149,7 @@ The keytool command does not allow importing an existing private key into a keys
         adb push <keystore name>.bks /data/user/0/your_app_dir_goes_here/files/<keystore name>
 
     The directory and filename used will depend on your use case.  Typically the application's files directory is in /data/user/0/<app namespace>/files/.  You may however choose to locate your keystore on removable media or another space on the filesystem.  The SDK allows for specifying the file path and name of the keystore so the choice is up to you.
+	
+## Using AWS IOT with user sign-in
+
+This sample demonstrates enabling guest users to use the AWS IoT APIs to securely publish-to and subscribe-from MQTT topics. Instructions to use AWS IoT with user sign-in can be found [here](./AuthenticatedPubsub.md)
```

Comment:
Does the sample demonstrate enabling guest users?

If not, you can add text here along the lines of " The sample demonstrates how to use IOT with certificate based authentication. If you'd like to instead use a signed-in user as the authentication mechanism, the steps are listed [here](./AuthenticatedPubsub.md)"